Makefile

```diff
@@ -8,7 +8,7 @@ test:
 # Command to run flake8 (code style check), isort (import ordering), and black (code formatting)
 # Subsequent commands still run if the previous fails, but return failure at the end
 checkstyle:
-	flake8 .; flake8_status=$$?; \
+	flake8 . --exclude=.venv; flake8_status=$$?; \
```

Comment:
can be added https://github.com/linkedin/Liger-Kernel/blob/main/.flake8, but why do you have venv?

Reply:
Added to .flake8. I'm managing multiple repos on same VM, so it's easier to keep separate venvs within each repo.